ranger/tmpl/error.tmpl

```diff
@@ -5,7 +5,7 @@
     <title>{{ metadata "title" }}</title>
     <meta name="description" content="{{ metadata "description" }}"> 
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="icon" sizes="48x48" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQAAAAEAAgMAAAAhHED1AAAACVBMVEUAAAAoUL633XlwfAKmAAAAAXRSTlMAQObYZgAAAz5JREFUeNrt203OozAQhOEGiQ377w4jTuEjeGHu46N4afmUI81IgxJ+3jalJItJ7fOILhuHSMS++eYbR9Y/iXYvP+uWYN2Z1sd0X/wuQfg8CPB5EODzIHD9t5oc1tMk9wDKENN6FWEA2NbQ4Ba+AEjEC6DQBWAiLAEH9gAnwASYBBVyYAJOhAoxCSbgwAScCGuASTABBybgBJgAE2ERMQkq4EAFnAAVYCJUgElQAUcGAnR4o4RBBaY+IEEFHBkI0CED0CEmqkCCRWAAOuQA0N3iqgPaMgwiwFfAy3AHEIcwdSuZKgS4o995LG6RgaF3HaFIBFhgQFgK/UHHVCHQ0x4DLDCgLEWkx34GQFAAE3YSFAkACAjwUjxE/xGrCkZxA7+gSNzLS4YiCWjNQECgQJEINCiSgXIu8N00tqtLGJxAgT2NQIMdyUABAYHWv6fTI5DhcDgH5r9AhcMBgZZBQKDA4YBAs6t4gAwCArX3nH4GGpzTZ8DyDyggIFDtOgg0ACYAcIYBAJ4BgSYDWQWqCrQuIBwAWQWKClQVaDeA9pCsAkUAsITBAzQZyCpQVKCqQJOBrAJFBaoAQAmTE8gqUFSgqkCTgfxxoPQBC7TIgLvFHzeQPw4UH2CnQFWBJgP540BRgXrzQXPLW4H5CMgqUN4BpCugqkD7PJDpq3EDxkOgfByoDMRLoH0eyG7AXgQUBMIOgGWY3gw0P7C8CMgE2KuB4gbme0AioL4PGNtxZAC+nCMC2QvYq4ByDQQdoK1YXwwYAs0NzHeA5AAyAL07aVWB3T7ijfBSIDiA6gaWG4DpQPdO2q2iDnQeavtV7DqTdGB4WkVYBgLMBRQ3MPcC6QkYJYCfk/ZAVIFg0CIDvhZPATNoEYB0B5j2HXKL+QQIe2BRgbkLMBFIBwD/dpvgBRAG4CWYpQMwgxIASIfAiAC9A0N30w+9B7S4ATvOSAC+AoPAfgKe4Qiws8wA8Es8PiCcAwsD2wS8DnuAX0MCYD8B15ifgWSX2QOdf5NbELDrjAREgywAGGW8BqJh5tMTxfu3sisgeIDxHIjmynwGJHNmOQSG1dw5BCzYHaHYvSwqYDMBnGU7lhWj2Dff/Bf5DdIZ5ke0sYEeAAAAAElFTkSuQmCC" />
+    <link rel="icon" sizes="48x48" href="{{ asset " favicon.ico" }}" />
```

Comment:
During QA I discovered that Safari still doesn't want to load this, so I'm just falling back to the project supported `favicon.ico` for now.  Safari v17 was released recently and has SVG favicon support in technical preview, so hopefully it will be resolved, and we can remove this entirely.